### ISSUE TITLE
Can we provide user statistic tracking on Primo? (overall analytics) 

### ISSUE THREAD

**Issue by tmiles2** [emory-libraries/blacklight-catalog#41]
Is there any way for us to extract which pages are visited most, which links are clicked most etc. (for the Nov 2019 - Feb 2020)

**Comment by laura-ake** [comment]
I have started this

**Comment by laura-ake** [comment]
I uploaded a dashboard file from Primo Analytics that I found while trying to clean out my personal folders, which will disappear when the OBI to OAS migration happens in Primo starting Sunday.  Take a look - might it fulfill some of this need?  

I believe Mark was also asking about most frequent searches.  Would the files I and Chris Pollette shared with SEND do the trick?  

[Primo Actions - Dashboard - Laura.xlsx](https://github.com/emory-libraries/blacklight-catalog/files/5518491/Primo.Actions.-.Dashboard.-.Laura.xlsx)
